hive/src/main/java/com/netease/arctic/hive/HiveTableProperties.java

```diff
@@ -48,6 +48,8 @@ public class HiveTableProperties {
 
   public static final String ALLOW_HIVE_TABLE_EXISTED = "allow-hive-table-existed";
 
+  public static final String WATERMARK_HIVE = "watermark.hive";
```

Comment:
Is this unused?

Reply:
Yes, both `watermark.base-store` and `watermark.hive` is unimplemented in this PR and will be done in future version. I would prefer to keep this property define here.